src/main/kotlin/no/nav/syfo/api/endpoints/ArbeidsuforhetEndpoints.kt

```diff
@@ -43,6 +44,39 @@ fun Route.registerArbeidsuforhetEndpoints(
             call.respond(HttpStatusCode.OK, responseDTO)
         }
 
+        post(vurderingPath) {
+            val requestDTO = call.receive<VurderingRequestDTO>()
+            if (requestDTO.begrunnelse.isBlank() || requestDTO.document.isEmpty()) {
```

Comment:
`AVSLAG` skal kanskje ikke ha document? Men kan evt tilpasses senere når vi har landet den journalføringsdiskusjonen :sweat_smile:

Reply:
Ja, det kan nok stemme at den faktisk burde være en tom liste for `AVSLAG` 🤔 Jeg skal prate med Ingrid i dag, og så kan vi fikse deretter 👍🏼